app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/SessionPropertiesFormatter.kt

```diff
@@ -0,0 +1,45 @@
+package nerd.tuxmobil.fahrplan.congress.utils
+
+import nerd.tuxmobil.fahrplan.congress.models.Session
+
+class SessionPropertiesFormatter {
+
+    fun getFormattedSpeakers(session: Session) =
+        with(session.speakers) {
+            when {
+                isEmpty() -> ""
+                else -> joinToString(", ")
+            }
+        }
```

Comment:
The previous code checked for `null`. If the value can never be `null`, this can simply be

```kotlin
fun getFormattedSpeakers(session: Session) = session.speakers.joinToString()
```

Otherwise
```kotlin
fun getFormattedSpeakers(session: Session) = session.speakers?.joinToString().orEmpty()
```

Reply:
:heavy_check_mark: Thank you. Changed.